actions/cherry_picker/functions.py

```diff
@@ -152,7 +152,8 @@ def run_cherry_pick(is_prod, commit_id, target_branch_name):
                 update_lockfile(changed_files, True)
             else:
                 subprocess.run(['git', 'cherry-pick', '--skip'])
-                raise Exception("Cherry-pick was attempted, but there may be merge conflict(s). Please resolve manually.\ncc: @bazelbuild/triage")
+                unmerged_files_names = ("`" + "`\n`".join(unmerged_all_files)).rstrip("`")
```

Comment:
```suggestion
                unmerged_files_names = "`" + "`\n`".join(unmerged_all_files) + "`"
```
I meant that you could add `.strip()` to line 147, before the `.split("\n")`.

Your version works too, but is brittle because if `unmerged_all_files` did _not_ end with an empty line, you could end up with an unclosed backtick.

Reply:
This works! Thanks